### ISSUE TITLE
Bitbucket - no update available

### ISSUE THREAD

**Issue by vyskoczilova** [YahnisElsts/plugin-update-checker#265]
Hi, how do I debug theme update? I have tried to implement it from private bitbucket, but somehow, no error is shown or update downloaded, do you see where the problem could be?

```
require 'vendor/plugin-update-checker/plugin-update-checker.php';
$myUpdateChecker = Puc_v4_Factory::buildUpdateChecker(
	'https://bitbucket.org/kybernaut/theme-isbjorn/',
	get_stylesheet_directory() . '/style.css',
	'isbjorn'
);

//Optional: If you're using a private repository, create an OAuth consumer
//and set the authentication credentials like this:
//Note: For now you need to check "This is a private consumer" when
//creating the consumer to work around #134:
// https://github.com/YahnisElsts/plugin-update-checker/issues/134
$myUpdateChecker->setAuthentication(array(
	'consumer_key' => 'myConsumerKey',
	'consumer_secret' => 'mySecretKey',
));

$myUpdateChecker->setBranch('master');
```

I have tried to release it via tag '1.0.0' and same time changed the theme's styles version to 1.0.0

**Comment by vyskoczilova** [comment]
fixed after wrote it down, there should be `get_template_directory` when it's used as a parent theme.
